affordance_primitives/include/affordance_primitives/task_estimator/task_estimator.hpp

```diff
@@ -57,8 +57,9 @@ class TaskEstimator
   /** Resets the internal estimation
    *
    * @param reset_val The value to reset to
+   * @return True if successful, false otherwise
    */
-  virtual void resetTaskEstimation(double reset_val = 0) = 0;
+  virtual bool resetTaskEstimation(double reset_val = 0) = 0;
```

Comment:
I'm being annoying picky. I know the internal representation is the same for `double 0` and `double 0.0` , however, Akita finds using the latter slightly better in communicating to another human being.